cli/commands/up.go

```diff
@@ -69,13 +71,22 @@ Run the postgres and localstack services directly:
 			if len(serviceNames) == 0 {
 				serviceNames = opts.serviceNames
 			}
+
+			serviceTags := make(map[string]string)
+			for _, serviceTag := range opts.serviceTags {
+				sb := strings.Split(serviceTag, ":")
+				// replacing forward slashes with dashes (because that's how ECR images are tagged)
+				trimmedBranch := strings.ReplaceAll(sb[1], "/", "-")
```

Comment:
This is too internal to how we happen to tag things in ECR in our own CI. I would take this out, and have people use the correct tags directly

Reply:
ok cool, i'll remove it then